### PR TITLE
feat(navigationItemTrigger): L3-4181 added scroll on dropdown

### PR DESCRIPTION
**Pull request by constantinehuzenko**
**Jira ticket**

[L3-4181](https://phillipsauctions.atlassian.net/browse/L3-4181)

**Screenshots**
| Before | After |
| ------ | ----- |
| image | image |

**Summary**

Added scroll to the auctions links

**Change List (describe the changes made to the files)**

- src/components/Navigation/NavigationItemTrigger/_navigationItemTrigger.scss: added scroll to the auction container
- src/site-furniture/Header/Header.stories.tsx: added more auctions to show scroll
- src/site-furniture/Header/Header.tsx: added prop which will help to get i18 link on Phillips-Public side

**Acceptance Test (how to verify the PR)**

- resize height of header and verify that scroll is presented in auction links
- verify that when auctions dropdown open there is some space left bellow

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-4181]: https://phillipsauctions.atlassian.net/browse/L3-4181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/site-furniture/Header/Header.stories.tsx
Comment (adietrich3074) [inline src/site-furniture/Header/Header.stories.tsx:1]: Thank you for updating the story to account for this 👍

File: src/site-furniture/Header/Header.tsx [hunk 1]
```diff
@@ -96,7 +101,7 @@ const Header = forwardRef<HTMLElement, HeaderProps>(
             <span /> {/** this is here so we can do transitions with pseudo icons */}
           </button>
           <h1 data-testid="header-logo" className={`${px}-header__logo`}>
-            <a href="/" aria-label={logoText}>
+            <a href={logoHref} aria-label={logoText}>
```
Comment (adietrich3074) [inline src/site-furniture/Header/Header.tsx:104]: I cant believe we didn't have this in here already 😆

File: src/components/Navigation/NavigationItemTrigger/_navigationItemTrigger.scss [hunk 2]
```diff
@@ -55,13 +55,20 @@
 
     &--hovered &__submenu {
       left: 0;
-      max-height: 700px;
+      max-height: calc(90vh - #{$header-height});
```
Comment (adietrich3074) [inline src/components/Navigation/NavigationItemTrigger/_navigationItemTrigger.scss:58]: Minor nitpick. Do we need the #{} around $header-height. The calc in the style above this doesn't need it so I'm just curious what this does for us.

**Review by sgrund14 — APPROVED** [review]

**Review by adietrich3074 — APPROVED** [review]
Looks good to me. I appreciate the updates to the story to reflect how the component reacts with the overflow content 👍

**Pull request merged**